### DISCUSSION TITLE
Auto-generated docs

### DISCUSSION THREAD

**Discussion by diligiant** [sbertix/Swiftagram#206] [General]
When I compare https://sbertix.github.io/Swiftagram to https://sbertix.github.io/ComposableRequest/ it looks like it.

**Comment by sbertix** [comment]
Hey @diligiant, 

I forgot to update the `README.md` hahaha
You can find **Swiftagram** docs [here](https://sbertix.github.io/Swiftagram/Swiftagram) and **SwiftagramCrypto**'s [here](https://sbertix.github.io/Swiftagram/SwiftagramCrypto).

**Reply by sbertix** [reply]
This is now fixed 💪 
Thank you for pointing it out @diligiant 😊
